tests/dashbio_demos/dash-alignment-viewer/app.py

```diff
@@ -552,7 +549,7 @@ def layout():
         ]),
         dcc.Loading(className='dashbio-loading', children=html.Div([
             dash_bio.AlignmentChart(
-                id='alignment-chart',
+                id='alignment-viewer',
```

Comment:
These should all be changed to `alignment-chart`. The name of the app is actually wrong -- it should be `app_alignment_chart.py` (or, in the new structure, `dash-alignment-chart/app.py`.

Reply:
Should I rename the app in the gallery as well?

Reply:
It should already be renamed! Looks like I must have forgotten to change the selectors and the app title that's displayed in the header -- my mistake. https://dash-gallery.plotly.host/dash-alignment-chart/

Reply:
Fixed!